src/embed.ts

```diff
@@ -8,6 +8,7 @@ import { Config as VlConfig, TopLevelSpec as VlSpec } from 'vega-lite';
 import schemaParser from 'vega-schema-url-parser';
 import * as themes from 'vega-themes';
 import { Handler, Options as TooltipOptions } from 'vega-tooltip';
+import { EncodeEntryName } from 'vega-typings';
```

Comment:
I just noticed this but let's add this to the Vega lib import above.

Reply:
Done!